python/pfs/instmodel/makePfsDesign.py

```diff
@@ -8,6 +8,24 @@
 SPECTROGRAPHS = [1, 2, 3, 4]  # Spectrograph numbers
 
 
+def holeToFiberId(holes, spectrograph):
+    """Convert hole numbers to fiberIds
+
+    Parameters
+    ----------
+    holes : array_like of `int`
+        Hole numbers (1..651).
+    spectrograph : `int`
+        Spectrograph number (1..4).
+
+    Returns
+    -------
+    fiberId : `numpy.ndarray` of `int`
+        Fiber identifiers.
+    """
+    return NUM_FIBERS*(spectrograph - 1) + np.array(holes)
+
+
```

Comment:
Should this be a pfs_utils function as it could be useful elsewhere (OK as a separate task)

Reply:
@CraigLoomis Would this be useful in pfs_utils?

Reply:
I'm also thinking it would be useful for Fabrice to have a related function that returns the ``(spectrograph, hole)`` from a given fiberId. Again, this can be part of a separate ticket.